lib/matplotlib/cbook/__init__.py

```diff
@@ -249,9 +249,12 @@ def __hash__(self):
         return self._hash
 
 
+def _excption_printer(exp):
```

Comment:
typo? (exception)
also I usually use "exc", not "exp", as variable name for an exception